site/about/style-guide.qmd

```diff
@@ -0,0 +1,230 @@
+---
+title: "ValidMind Style Guide"
```

Comment:
I'm pretty sure our style guide says to use sentence-style capitalization. 😁 

```suggestion
title: "ValidMind style guide"
```

Reply:
Haha, good point!